packages/shared/src/lib/log.ts

```diff
@@ -170,6 +170,7 @@ export enum LogEvent {
   ScheduleStreakReminder = 'schedule streak reminder',
   StreakRecover = 'restore streak',
   DismissStreakRecover = 'dimiss streaks milestone',
+  StreakTimezoneMismatch = 'streak timezone mismatch',
```

Comment:
What's the difference between this and the click with "streak timezone mismatch prompt"?

Reply:
@idoshamun this is (currently) a once off to see how many users will even see the warning, it is once of for now to not spam https://github.com/dailydotdev/apps/pull/4071#discussion_r1918594389